### DISCUSSION TITLE
is deconttX over correct my data

### DISCUSSION THREAD

**Discussion by mbourgey** [campbio/celda#317] [Q&A]
Hello, 

I ran DecontX on my PBMC data. Overall it works well except for one population the pDC. This population is very special in PBMC data and always looks likes a mix of different population. Thus after decontX the pDC now seems to cluster with the Bcells 
So my question are: 
Can we exclude a set of cell in the decontX analysis ? 
Is there a way to includes additional marker/cluster in the decontX process in order to consider this population, by looking at the documentation, I was thinking  of using the z paramter to use know cluster. in that case what level of subpopulation will be the best to use ?

Thank you in advance

Mathieu

**Comment by joshua-d-campbell** [comment]
Hi @mbourgey,
Thanks for trying out our tool. There are two things you can try: 1) Manually adjust the "z" label (i.e. cell cluster labels) to make sure that your pDCs are in in their own cluster. If they are currently clustered with the B-cells, the decontX will be removing transcripts more related to other populations (including other dendritic cells and myeloid cells). 2) The other option is that you can put your pDCs, B-cells, and the other population that your pDCs look like (other dendritic cells or other myeloid cells?) all into 1 group. Thus counts from B-cells and this other cell population will not be subtracted from your pDCs. 

Hope that helps.

**Comment by joshua-d-campbell** [comment]
I will also be converting this to a Discussion since it is more about how to best run the tool in certain circumstances.

**Reply by mbourgey** [reply]
Thank you for your answer I will try these 2 solutions and let you know. 

Mathieu
